### ISSUE TITLE
Block the Spot saying versions are incompatible

### ISSUE THREAD

**Issue by silverleaf321** [Nuzair46/BlockTheSpot-Mac#84]
### 🧐 Have you fully read the README and viewed similar issues?

- [X] I have read the README
- [X] I have checked older issues, open and closed
- [X] I did a clean install of Spotify and BlockTheSpot

### Сountry of your account

United States

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

Yes

### ℹ Computer information

```markdown
- Spotify: Spotify for macOS (Apple Silicon)
1.2.49.439.gfae492c7
- macOS: macOS Sonoma 14.6
- Shell: bash
```

### 📝 Description

1. Uninstall and reinstall spotify from the spotify website
2. Run the install command copy pasted from the Readme

### 🖥️ Terminal Input/Output

awk: towc: multibyte conversion failure on: '?# '

 input record number 1, file /Applications/Spotify.app/Contents/Info.plist
 source line number 1

************************
BlockTheSpot-Mac by @Nuzair46
************************

Spotify version: 
BlockTheSpot-Mac version: 1.2.32.985.g3be2709c

This version of BlockTheSpot-Mac is not compatible with your Spotify version.
Please use an older version of BlockTheSpot-Mac or update Spotify.

**Comment by arjun-com** [comment]
+ 1 same for me too

**Comment by coltenthefirst** [comment]
+1

**Comment by shellwhale** [comment]
Same, I noticed that the Spotify version does not show up on the infos

**Comment by asadnwfp** [comment]
+1 
**Specs** 
```
- Spotify : Spotify for macOS (Intel)
1.2.50.335.g5e2860a8
- macOS: macOS Sequoia 15.1
- Shell: bash
```

**Comment by SkyMarsh14** [comment]
Same here.
```
- Spotify for macOS (Apple Silicon)
1.2.51.345.gcc39d911
- masOS: Sequoia 15.1.1
- Shell: bash
```

**Comment by DrafBear** [comment]
SpotX-Bash doesn't work for me as of today 

What I ended up doing is downgrading my spotify version (you can go to [spotify's website](https://www.spotify.com/de-en/download/other/) and download version 10.15). Once you download the old version of spotify, just run the install command in terminal again and it should work fine

**Comment by alexshibu1** [comment]
I tried it with the old version. it still has ads.

**Comment by DrafBear** [comment]
Forgot to update this, the **10.13** version is what worked for me, _not_ the 10.15. So far it has worked without any issues

**Comment by DrafBear** [comment]
Though I haven't bothered trying 10.14, you could give that a try as well

**Comment by billy0402** [comment]
The download OS version to Spotify version:
**Newest**: 1.2.52.442.g01893f92
**10.15**: 1.2.37.701.ge66eb7bc
**10.13-14**: 1.2.20.1216.ge7a7b92f
**10.11-12**: 1.1.89.862.g94554d24

The BlockTheSpot-Mac version is currently supported up to 1.2.32.985.g3be2709c.
So 10.15 or newer won't work.

**Comment by Nuzair46** [comment]
should be fixed with #83 #85
